action.yml

```diff
@@ -70,7 +69,7 @@ runs:
         echo "FINAL_TEXT=$FINAL_TEXT" >> "$GITHUB_OUTPUT"
     - name: Post to a Slack channel
       id: slack
-      if: inputs.slack-webhook-url != ''
+      if: ${{ inputs.slack-webhook-url != '' && inputs.channel != '' }}
```

Comment:
Hmm, is empty string the default when no parameter is passed in?

Reply:
we confirmed this async, but I'll also paste the issue you share :upside_down_face: 

https://github.com/actions/toolkit/issues/940

Reply:
This bug seems to indicate that it is: https://github.com/actions/toolkit/issues/940
Lots of examples on the internet corroborate this pattern as well, though I was unable to find official docs on this